src/__snapshots__/Filter.test.js.snap

```diff
@@ -0,0 +1,52 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Filter render with snapshot 1`] = `
+<form
+  className="filter-section"
```

Comment:
It's great to see our snapshots are updating. Thanks Kim for testing snapshot functionality with me.